From: Guilhem Moulin <guilhem@debian.org>
Date: Fri, 15 Jul 2022 12:16:05 +0200
Subject: localtest: Exclude `ssh-plugin-test` when $RUN_SSH_PLUGIN_TEST is
 empty

We want to use TESTSUITE_NOSKIP=y to keep full test coverage, but run
`ssh-plugin-test` in a separate autopkgtest because it has since it has
its own dependency set.  So we need `make -f Makefile.localtest tests
TESTSUITE_NOSKIP=y` to filter out `ssh-plugin-test` instead of marking
it as skipped.

Forwarded: not-needed
---
 tests/Makefile.localtest | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tests/Makefile.localtest b/tests/Makefile.localtest
index fae3e28..3f58ff6 100644
--- a/tests/Makefile.localtest
+++ b/tests/Makefile.localtest
@@ -8,6 +8,10 @@ CFLAGS=-O2 -g -Wall
 LDLIBS=-lcryptsetup -ldevmapper
 TESTS=$(wildcard *-test *-test2) api-test api-test-2 all-symbols-test
 
+ifeq (,$(RUN_SSH_PLUGIN_TEST))
+TESTS:=$(filter-out ssh-plugin-test, $(TESTS))
+endif
+
 differ: differ.o
 	$(CC) -o $@ $^
 
